### PR TITLE
Bug Fix in Education Container

### PR DESCRIPTION
**Pull request by NajamShehzad**
I was unable to toggle the "education" section

### CODE REVIEW

**Review by saadpasta — COMMENTED** [review]
@NajamShehzad Thanks for contributing to DeveloperFolio.
Can you please provide the description for this PR, I really don't get it. Thanks ✌

**Comment by NajamShehzad** [comment]
Sure, It was not hiding the education section if we're passing "viewEducation:false". I've also found some more bugs like the contact us section if I get "isHirable:false" then it's still showing "open for opportunities Yes"
Another one the items in "Big Project" are not aligned

That's it 😄

**Comment by kartikcho** [comment]
> It was not hiding the education section if we're passing "viewEducation:false"

Hmm weird that it got through, nice catch! 

> the contact us section if I get "isHirable:false" then it's still showing "open for opportunities Yes"

Could you open an issue with more context on this?

>  the items in "Big Project" are not aligned

I'll open an issue for this, the whole section could use a revamp!

File: src/containers/education/Education.js [hunk 1]
```diff
@@ -4,14 +4,17 @@ import EducationCard from '../../components/educationCard/EducationCard';
...
-    <div className="education-section" id="education">
-        <h1 className="education-heading">Education</h1>
-        <div className="education-card-container">
-          {educationInfo.schools.map((school) => (
-            <EducationCard school={school} />
-          ))}
-        </div>
-    </div>
-  );
+   if(educationInfo.viewEducation){
```
Comment (kartikcho) [inline src/containers/education/Education.js:7]: Avoid formatting as it shows changes in lines where there are none which can be confusing while reviewing. It isn't a problem for this small PR but can be really difficult in big ones.

**Review by kartikcho — APPROVED** [review]
The code looks good, thanks for providing this fix!

**Review by naveen521kk — APPROVED** [review]
LGTM! Please follow what @kartikcho asked.

**Comment by kartikcho** [comment]
Approved it as is for now, will run a prettier script in a later PR.

**Comment by naveen521kk** [comment]
@all-contributors please add @NajamShehzad for code

**Pull request merged**